deck/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Deck(models.Model):
+    heros = ['druid', 'hunter', 'mage',
```

Comment:
Well its defined here, and we could reuse it from here. Its not really defined elsewhere